Documentation/github-connector.md

```diff
@@ -8,7 +8,7 @@ When a client redeems a refresh token through dex, dex will re-query GitHub to u
 
 ## Caveats
 
-* Please note that in order for a user to be authenticated via GitHub, the user needs to mark their email id as public on GitHub. This will enable the API to return the user's email to Dex.
+* A user added to an organization __after__ they have granted an application access their GitHub information via dex will not be redirected to a 'request organization access' page. That user should log into their GitHub account, go to Settings -> Authorized OAuth Apps, click 'Revoke' to revoke the application's grant, and restart the dex login process. Dex will issue a new grant to the application with proper organization permissions. This issue arises because GitHub's OAuth redirect URI does not return a 'request organization access' page.
```

Comment:
I don't think this doc's change should go in as a part of this PR. The title of the PR and the above documentation don't have any connection. It can be confusing. For this PR just remove the line about the email id of the user having to be public